src/pages/Movies/styled.jsx

```diff
@@ -1,92 +1,103 @@
 import styled from "styled-components";
 import { ReactComponent as LeftIcon } from "../../assets/icons/left.svg";
 import { ReactComponent as RightIcon } from "../../assets/icons/right.svg";
+import { NavLink } from "react-router-dom";
+
+export const StyledNav = styled(NavLink)`
+  text-decoration: none;
+  color: ${({ theme }) => theme.colors.site.text};
+  &:hover {
+    filter: brightness(130%); //demo
```

Comment:
In my opinion, 130% is too much. It transforms movie image in a weird way. Try lowering the percentage of brightness or think abour another way of hover effect. Box shadow looks great. I think scale transform would go well on hover. Maybe about 110%. You can also add duration of transform effect to make it more smooth.

Reply:
you were right, now after making these corrections it looks great